src/score.js

```diff
@@ -101,8 +97,9 @@ export const makeCandidateScoreCalc = (config, hostname, chatDomain, apiSlug, ap
         if (isAskingForOtherUser) {
             userId = +(content.match(/(\d+)(?:\?|$)/)[1]);
         }
+
         // If not mod and not Chat.SO, resolve election site user id from requestor's chat id (chat has different ids)
-        else if (!isSO) {
+        if (!isSO) {
```

Comment:
I just realised this, can't remove the else here - since the id that the mod provides in the if-statement above is not the user's chat id